test/types.d.ts

```diff
@@ -45,17 +43,16 @@ declare const browserType: import('../index').BrowserType<import('../index').Bro
 
 // global variables in assets
 declare const testOptions: {
-    FFOX: boolean;
-    WEBKIT: boolean;
-    CHROMIUM: boolean;
-    MAC: boolean;
-    LINUX: boolean;
-    WIN: boolean;
     HEADLESS: boolean;
     WIRE: boolean;
 };
 
-declare const testPath : string;
+declare var MAC: boolean;
```

Comment:
nit: declare this in base.fixture where they are defined?

Reply:
I failed to :-(